papers/scipy/scipy.rst

```diff
@@ -26,36 +18,56 @@
     same size as and side-by-side with the z.  Using `a_{_N}` makes it
     very clear that N is a subscript of a.
 
------
-SciPy
------
+-------------------------------------
+Discrete-time Linear Filters in SciPy
+-------------------------------------
 
 .. class:: abstract
 
 The SciPy_ library is one of the core packages of the PyData stack.  It
 includes modules for statistics, optimization, interpolation, integration,
 linear algebra, Fourier transforms, signal and image processing, ODE solvers,
 special functions, sparse matrices, and more.
+In this chapter, we demonstrate many of the tools provided by the ``signal``
+subpackage of the SciPy library for the design and analysis of linear
+filters for discrete-time signals, including filter representation,
+frequency response computation and optimal FIR filter design.
+
 
 
 .. _SciPy: http://scipy.org/scipylib/index.html
 
 .. class:: keywords
 
-algorithms, optimization, statistics, linear algebra, signal processing,
-sparse matrix, interpolation, numerical integration, special functions
+algorithms, signal processing,  IIR filter, FIR filter
 
 .. contents::
 
 Introduction
 ============
+The SciPy_ library, created in 2001, is one of the core packages of the
+PyData stack.  It includes modules for statistics, optimization,
+interpolation, integration, linear algebra, Fourier transforms, signal
+and image processing, ODE solvers, special functions, sparse matrices,
+and more.
+
+The ``signal`` subpackage within the SciPy library includes tools
+for several areas of computation, including signal processing, interpolation,
+linear systems analysis and even some elementary image processing.  In this
+Cookbook chapter, we focus on specific subset of the capabilities of
+of this subpackage: the design and analysis of linear filters for discrete-time
+signals.
```

Comment:
Maybe add a sentence here like: "These filters are commonly used in digital signal processing (DSP) for audio signals, as well as in <control systems, etc.> ..."

Reply:
Good idea.  I went with "These filters are commonly used in digital signal processing (DSP) for audio signals and other time series data with a fixed sample rate."